### PR TITLE
fix juniper mic-3d-16xge-sfpp

### PR DESCRIPTION
**Pull request by gpipito**
fixed model value in the yaml file

### CODE REVIEW

**Comment by danner26** [comment]
Hey @gpipito it looks like there is a conflict. Please resolve that and let me know when this is ready to be reviewed. Thanks!

File: device-types/Juniper/PTX10001-36MR.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Juniper
+model: PTX10001-36MR
+slug: ptx10001-36mr
```
Comment (danner26) [inline device-types/Juniper/PTX10001-36MR.yaml:4]: Please append the manufacturer name to the device like so: `juniper-`

File: device-types/Juniper/PTX10001-36MR.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+interfaces:
```
Comment (danner26) [inline device-types/Juniper/PTX10001-36MR.yaml:5]: You are missing a ton of information to make this a complete and valid PR. Please add the following data points: 

- `part_number`
- `u_height`
- `is_full_depth`
- `airflow`
- `weight` and `weight_unit`
- `comments` with a makrdown formatted link to the devices datasheet (on the manufacturer website, if available)

File: device-types/Juniper/PTX10001-36MR.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - name: re0:mgmt-0
+    type: 1000base-t
+    mgmt_only: true
+  - name: et-0/0/0
```
Comment (danner26) [inline device-types/Juniper/PTX10001-36MR.yaml:9]: Please confirm for me these are the full interface names. For instance, on a Cisco 2960 you will see `GigabitEthernet1/0/1` but you can shorthand it to `gi1/0/1`. We opt for the most complete information so we expect the interface to be named exactly as it is shown.

Can you please confirm that the devices configuration shows `et-0/0/0`?

File: device-types/Juniper/PTX10001-36MR.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    type: 100gbase-x-qsfp28
+  - name: et-0/2/11
+    type: 100gbase-x-qsfp28
+power-ports:
+  - name: PSM0
+    type: iec-60320-c20
+    maximum_draw: 3000
+  - name: PSM1
+    type: iec-60320-c20
+    maximum_draw: 3000
```
Comment (danner26) [inline device-types/Juniper/PTX10001-36MR.yaml:85]: It appears this devices PSUs are modular. Thus, you should have these defined under `module-bays` and the applicable PSU modules should be added to the module-types folder if they do not already exist
It also appears this device has 6 modular fans. Those should be added to `module-bays` as well

**Review by danner26 — CHANGES_REQUESTED** [review]
Please review and fix requested changes

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 1 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:4]: sorry, I don't get it. Where am I supposed to write "juniper-"?
The other juniper in the repo are reported like this:

---
manufacturer: Juniper
model: QFX5110-48S-4C
slug: qfx5110-48s-4c
interfaces:
...

---
manufacturer: Juniper
model: MX480
slug: mx480
u_height: 8
is_full_depth: true
...

Consider that I used the QFX file as a template for my ptx.

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 2 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:5]: ok...but again a lot of device in the repo are missing that information too. I'm just copy/pasting.

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 3 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:9]: that is Juniper naming convention:

image

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 4 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:85]: I can remove the power-ports information. Not a problem.
Again however, a lot of juniper device do not report that level of detail.

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 1 above)
Comment (danner26) [reply device-types/Juniper/PTX10001-36MR.yaml:4]: That means they havent been migrated yet, one moment ill take care of that now

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 4 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:85]: if you look at MX104-BASE or MX104-PREMIUM for example, you'll see they report PEM modules as built-in power-ports.

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 2 above)
Comment (danner26) [reply device-types/Juniper/PTX10001-36MR.yaml:5]: I understand, new PRs require the full set of information to be added. A lot of these files are years old before these data points existed in NetBox. All new PRs require as much information as possible

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 3 above)
Comment (danner26) [reply device-types/Juniper/PTX10001-36MR.yaml:9]: Perfect! Thank you

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 1 above)
Comment (danner26) [reply device-types/Juniper/PTX10001-36MR.yaml:4]: @gpipito please run a git pull and take a look at the other Juniper devices now. You should see the new slug naming convention

File: device-types/Juniper/PTX10001-36MR.yaml (hunk 2 above)
Comment (gpipito) [reply device-types/Juniper/PTX10001-36MR.yaml:5]: ok. I'm going to find that information. However the chassis does not have a part number:

image

**Comment by gpipito** [comment]
I deleted PTX and PSM modules. Only the correction on mic model is left.

**Review by danner26 — APPROVED** [review]

**Comment by danner26** [comment]
Please feel free to open another PR for any other devices if you would like

**Pull request merged**